### PR TITLE
Add SyncProxyRulesCount as a metric

### PR DESCRIPTION
**Pull request by rramkumar1**
This PR adds a metric to kube proxy called SyncProxyRulesCount. This metric will allow us to identify situations in which kube proxy is failing to sync iptables rules. 

Release Note:
```release-note
None
```

### CODE REVIEW

File: pkg/proxy/winkernel/metrics.go [hunk 1]
```diff
@@ -34,13 +35,23 @@ var (
 			Buckets:   prometheus.ExponentialBuckets(1000, 2, 15),
 		},
 	)
+
+	// SyncProxyRulesCount is the number of times kube-proxy has synced proxy rules.
+	SyncProxyRulesCount = prometheus.NewCounter(
```
Comment (lavalamp) [inline pkg/proxy/winkernel/metrics.go]: Why is this needed? Histogram already tracks count: https://prometheus.io/docs/practices/histograms/#count-and-sum-of-observations

Comment (rramkumar1) [reply pkg/proxy/winkernel/metrics.go]: Ack, did not know about that. Will go ahead and scrap this PR
